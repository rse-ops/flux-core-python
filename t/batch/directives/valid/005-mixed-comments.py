#!/usr/bin/env python3
"""
flux: --nodes=4

Set an arbitrary value in jobspec:
flux: --setattr=user.foo="hello, earth"
"""
