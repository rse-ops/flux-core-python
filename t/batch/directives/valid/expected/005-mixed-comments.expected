SETARGS(['--nodes=4'])
SETARGS(['--setattr=user.foo=hello, earth'])
